README.md

```diff
@@ -1,24 +1,19 @@
-Warning: This repo is under active development and not yet suitable for use.
-============================================================================
-
 Ansible Role for Cloud Ops
 ==========================
 
 This Ansible role installs the Cloud Ops agents.
 
 Install this directory in your roles path (usually in a `roles` directory
-alongside your playbook) under the name `cloud_ops`:
+alongside your playbook) under the name `google_cloud_ops_agents`:
```

Comment:
Can we rebase off master to update the diff? These changes were merged by master by https://github.com/GoogleCloudPlatform/google-cloud-ops-agents-ansible/pull/50/files already it seems.

Reply:
Done.